src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -42,6 +44,129 @@
 public final class RqMultipartTest {
 
     /**
+     * Carriage return constant.
+     */
+    private static final String CR = "\r\n";
+    /**
+     * Two RqMultipart objects are equal when inited with equal Request objects.
+     * @throws IOException if some problem inside
+     */
+    @Test
+    public void equalsReturnsTrue() throws IOException {
```

Comment:
@ekondrashev see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names for test name and test description.